Library/ViewModels/AddNewCardViewModelTests.swift

```diff
@@ -405,9 +402,53 @@ internal final class AddNewCardViewModelTests: TestCase {
   func testReusableCardSwitchisOnByDefault_Settings() {
     self.rememberThisCardToggleViewControllerIsOn.assertDidNotEmitValue()
 
-    self.vm.inputs.configure(with: .settings)
+    self.vm.inputs.configure(with: .settings, project: nil)
     self.vm.inputs.viewDidLoad()
 
     self.rememberThisCardToggleViewControllerIsOn.assertValues([true])
   }
+
+  func testUnsupportedCardBrandsError() {
+    let project = Project.cosmicSurgery
+      |> Project.lens.availableCardTypes .~ [
+        "AMEX",
+        "MASTERCARD",
+        "VISA",
+        "UNION_PAY"
+      ]
+
+    self.vm.inputs.viewDidLoad()
+    self.vm.inputs.configure(with: .pledge, project: project)
+
+    self.creditCardValidationErrorContainerHidden
+      .assertValues([true], "Unsupported card message is hidden on viewDidLoad")
+
+    self.vm.inputs.creditCardChanged(cardDetails: ("3566", nil, nil, nil, .jcb))
+
+    self.creditCardValidationErrorContainerHidden
+      .assertValues([true, false], "Unsupported card message hides with a valid card brand")
```

Comment:
I think this message is incorrect. Shouldn't this be "Unsupported card message shows with invalid card brand" 🤔